packaging/prerelease/build_app.sh

```diff
@@ -12,6 +12,7 @@ nopull=${NOPULL:-} # Don't git pull
 client_commit=${CLIENT_COMMIT:-} # Commit hash on client to build from
 kbfs_commit=${KBFS_COMMIT:-} # Commit hash on kbfs to build from
 bucket_name=${BUCKET_NAME:-"prerelease.keybase.io"}
+s3host=${S3HOST:-"https://prerelease.keybase.io"}
```

Comment:
It looks like this will break if `BUCKET_NAME` is set but `S3HOST` and `TEST` are not?